auth/src/test/kotlin/com/tidal/sdk/auth/TokenRepositoryTest.kt

```diff
@@ -755,4 +756,31 @@ class TokenRepositoryTest {
             "No calls to the backend should have been made"
         }
     }
+
+    @Test
+    fun `there can only be one refresh call even when requested from multiple threads`() = runTest {
```

Comment:
There is another test that this one most likely replaces: 
```
`A call to getCredentials should not trigger a backend call if one is already in progress`
```
That test should now be removed I guess.

Reply:
Updated